docs/user/_index.md

```diff
@@ -80,9 +60,30 @@ machine as the Grafana Agent process. It:
 If the filter passes, the target is allowed to be scraped. Otherwise, the target
```

Comment:
'...the target can be scraped'? Or 'If the filter passes, the Agent can scrape the target'?

Reply:
I'll double check